schemas/2016-12-01/Microsoft.RecoveryServices.Backup.json

```diff
@@ -0,0 +1,1031 @@
+{
```

Comment:
Please could you remove `schemas/2016-12-01/Microsoft.RecoveryServices.json` as this file duplicates those resource types.